invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/factory.py

```diff
@@ -5,9 +5,10 @@
 import os
 import sys
 
+from invenio_config import create_config_loader
```

Comment:
This one is actually sorted correctly since it's a template - i.e. once the file has been used to generate a module it is correct inside that module. However I assume you have isort reporting that it's failing?